doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md

```diff
@@ -1305,19 +1379,23 @@ The security server system parameters are:
 
 To upload the configuration anchor, follow these steps.
 
-1.  On the **Configuration** menu, select **System Parameters**. The system parameters view is opened.
+1.  In the **Navigation tabs**, select **Settings**.
+
+2.  In the opening view select **System Parameters** tab.
 
-2.  In the **Configuration Anchor** section, click **Upload**.
+3.  In the **Configuration Anchor** section, click **UPLOAD**.
 
-3.  Find the anchor file from the local file system and click **Upload**.
+4.  Find the anchor file from the local file system and click **Upload**.
```

Comment:
```suggestion
4.  Click **Upload** and find the anchor file from the local file system.
```

Reply:
The file browser itself might have different verbs in different operating systems. Changed it to `Open` as it is in OSX and IIRC in Windows too